src/Jackett.Common/Definitions/amigosshare.yml

```diff
@@ -173,6 +173,9 @@
       _type:
         selector: div.list-group-item-content p.m-0 span.badge-info:contains("Rip"), div.list-group-item-content p.m-0 span.badge-info:contains("WEB-"), div.list-group-item-content p.m-0 span.badge-info:contains("TV"), div.list-group-item-content p.m-0 span.badge-info:contains("Blu-Ray"), div.list-group-item-content p.m-0 span.badge-info:contains("BD50"), div.list-group-item-content p.m-0 span.badge-info:contains("MUX"), div.list-group-item-content p.m-0 span.badge-info:contains("DVD"), div.list-group-item-content p.m-0 span.badge-info:contains("320"), div.list-group-item-content p.m-0 span.badge-info:contains("CAM"), div.list-group-item-content p.m-0 span.badge-info:contains("rip")
         optional: true
+      _language:
+        selector: div.list-group-item-content p.m-0 span.badge-primary[style$="#b6249d;"]
+        optional: true
```

Comment:
was it your intention to add `Original` or `Legendado` to the end of the other titles?